releaseNotes/AzureSpringCloudAppConfigurationConfig.md

```diff
@@ -0,0 +1,37 @@
+# azure-spring-cloud-appconfiguration-config
+
+[Source code][source_code] | [Package (Maven)][package] | [Product documentation][docs] | [Samples][samples]
+
+# azure-spring-cloud-appconfiguration-config-web
+
+[Source code web ][source_code_web] | [Package (Maven) web][package_web] | [Product documentation][docs]
+
+## 2.0.0-beta.1 - May 5, 2021
+
+* Enables loading of configurations from multiple config stores.
+  * Can load configurations by using key/label `selects`. Many `selects` can be used, default is key = `/applicaiton/*` with label = `${spring.profiles.active}` or if null `\0`
+  * Authentication vis Connection String, Managed Identity, or any method supported by Azure Identity SDK, see Token Credential Provider bellow.
```

Comment:
'bellow' is misspelled. I'm a bit confused by this bullet. Is this bullet advertising new authentication support? If it is it seems like it should be a standalone bullet.